### PR TITLE
Exception when matching include and exclude parameters

### PR DESCRIPTION
**Pull request by sblondon**
This PR is linked to https://github.com/leapfrogonline/rstr/issues/4.

I run the tests with `tox` on python 3.9 only.

I can add commits to improve the message by including the characters raising the exception if you think it's necessary.

### CODE REVIEW

File: rstr/rstr_base.py [hunk 1]
```diff
@@ -157,3 +160,7 @@ def __init__(self, _random=random, **alphabets):
         super(Rstr, self).__init__(_random=_random, **alphabets)
 
 default_instance = Rstr()
+
+
+class SameCharacterError(Exception):
+    pass
```
Comment (bjmc) [inline rstr/rstr_base.py:166]: Newline here please. :slightly_smiling_face:

Comment (bjmc) [reply rstr/rstr_base.py:166]: Inherit `SameCharacterError` from `ValueError` because it's a problem with the values passed.

**Comment by bjmc** [comment]
You're right, it would be a little nicer if it reported what the overlapping characters were in the error message. Do you want to add that?

File: rstr/rstr_base.py (hunk 1 above)
Comment (sblondon) [reply rstr/rstr_base.py]: I fixed them in a new commit.

**Comment by sblondon** [comment]
Yes, I will improve the message.

**Comment by sblondon** [comment]
It's commited. You can review the patches.

**Pull request merged**